### PR TITLE
Feature/rev 1888 shopify

### PR DESCRIPTION
**Pull request by aaditi-thakkar**
## What

* Add a new stream based on the original "orders" stream which also pulls in shop_id
## How

* "Orders" stream in shopify was subclassed to create a new stream named "ltk_orders" with an additional shop_id datapoint
## User Impact

shop_id will now be available through ltk_orders stream. 

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -65,4 +65,4 @@
 /airbyte-integrations/connectors/source-hubspot/ @airbytehq/python-team
 /airbyte-integrations/connectors/source-salesforce/ @airbytehq/python-team
 /airbyte-integrations/connectors/source-shopify/ @airbytehq/python-team
-/airbyte-integrations/connectors/source-stripe/ @airbytehq/python-team
+/airbyte-integrations/connectors/source-stripe/ @airbytehq/python-team
```
Comment (tjgalinis) [inline .github/CODEOWNERS:68]: adding a newline here would remove this file from the diff

File: airbyte-integrations/connectors/source-shopify/source_shopify/constants.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ADVERTISERS_QUERY = "SELECT advertiser_homepage, affiliateId \
                     FROM `rewardstyle`.`Advertisers` \
                     WHERE SOURCE = 39 and active = 1 \
-                    ORDER BY `name`;"
+                    ORDER BY `name` LIMIT 3;"
```
Comment (tjgalinis) [inline airbyte-integrations/connectors/source-shopify/source_shopify/constants.py:4]: only returning 3 advertisers? sorting by name may be problematic if we onboard, say, a brand called "Acme"...

I think sorting by ID would keep the stream consistent

File: airbyte-integrations/connectors/source-shopify/source_shopify/source.py [hunk 3]
```diff
@@ -254,7 +256,7 @@ def shops(self, shops):
     
     def gatherLTKShopifyStores(self, logger: logging.Logger):
         shops = []
-        shopify_stores = self._db_client._get_shopify_store_info()
+        shopify_stores = [{"advertiser_homepage": "igkhair.myshopify.com", "affiliateId": "2311487532"}] #self._db_client._get_shopify_store_info()
```
Comment (tjgalinis) [inline airbyte-integrations/connectors/source-shopify/source_shopify/source.py:259]: Hard-coding store info? Because there's no data in QA?

**Review by tjgalinis — APPROVED** [review]

File: airbyte-integrations/connectors/source-shopify/source_shopify/constants.py (hunk 2 above)
Comment (aaditi-thakkar) [reply airbyte-integrations/connectors/source-shopify/source_shopify/constants.py]: The limit was part of a rebase gone wrong. I have fixed it. Updated the ordering per ID as well.

**Pull request merged**